ufo/user.py

```diff
@@ -139,9 +144,13 @@ def add_user():
 @setup_required
 def user_details(user_id):
   user = models.User.query.get_or_404(user_id)
-  return flask.render_template('user_details.html',
-                               user=user,
-                               invite_code=_MakeInviteCode(user))
+  invite_code = _make_invite_code(user)
+  if invite_code is None:
+    return flask.render_template('user_details.html', user=user)
+  else:
+    invite_url = INVITE_CODE_URL_PREFIX + invite_code
```

Comment:
Oops, yeah that was supposed to be in the prefix constant, but I think I accidentally undo-ed one too many times and removed it. Added back now.